components/home/HomeBanner.tsx

```
@@ -0,0 +1,48 @@
+  buttonText,
+  buttonLink = "/login",
+  imageSrc,
+  isFooter,
+}: HomeBannerProps) => {
+  return (
+    <section className="box-border h-[540px] w-full bg-[#cfe5ff] sm:h-[540px]">
+      <figure className="mx-auto flex h-full items-end justify-center">
+        <div className="mb-[90px] max-w-[400px]">
+          <h1
+            className="mb-[4rem] max-w-[35rem] whitespace-normal break-keep text-center text-[4rem] font-bold leading-[5.6rem] sm:max-w-[24rem] sm:text-center sm:text-[3.2rem] sm:leading-[4rem] md:max-w-full"
+            dangerouslySetInnerHTML={{ __html: title }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)